include/mbedtls/x509_crt.h

```diff
@@ -190,7 +190,9 @@ mbedtls_x509_subject_alternative_name;
 typedef struct mbedtls_x509_crt_profile
 {
     uint32_t allowed_mds;       /**< MDs for signatures         */
-    uint32_t allowed_pks;       /**< PK algs for signatures     */
+    uint32_t allowed_pks;       /**< PK algs for public keys;
+                                 *   this applies to any CRT
```

Comment:
"any" cert in the change, or "all" certs in the chain?

(and for some reason I read a standalone "CRT" as "Chinese Remainder Theorem, but that's just me; I'm used to "cert" as an abbreviation for "certificate")

Reply:
Good point, that's "all". Also, as a matter of principle we shouldn't be using abbreviations in user-facing documentation.

(I'll note that there is some context: the file has crt in the name, and so does the structure, but if it tripped you then it might trip someone else, so let's avoid any ambiguity.)

Reply:
> the file has crt in the name, and so does the structure

At the risk of sounding defensive or just plain weird, it was the capitalisation of "CRT" more than anything that tripped me up! And I note that `MD`s and `PK`s are all uppercase, so it's even consistent with that, but thank you for changing it anyway - I've approved the PR

Reply:
No, I agree, for me as well the capitalization makes it more likely to interpret it as the initials of Chinese Remainder Theorem rather than the shortening/devowelisation of the work CeRTificate.